scripts/performance/configs/qwen_vl/qwen35_vl_workload_base_configs.py

```
@@ -0,0 +1,470 @@
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_B300_BF16",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_B300_FP8_CS",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_B300_FP8_MX",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_GB200_BF16",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_GB200_FP8_CS",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_GB200_FP8_MX",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_B200_BF16",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_B200_FP8_CS",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_B200_FP8_MX",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_H100_BF16",
+    "QWEN35_VL_397B_A17B_PRETRAIN_CONFIG_H100_FP8_CS",
+]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)